Felienne/hedy
# [UI FIX] Improves teachers welcome mail template

**Description**
In this PR we make a few small improvements to the teachers welcome mail template. Currently there are no newlines in the mailing making it hard to read and doesn't look really nice. We fix this in the .po files for en/nl/fr as these are currently already translated. Weblate should automatically copy the English change to all other languages. (Correct we if I'm wrong @Mark-Giesen).

**Fixes**
This PR fixes #2576.

**How to test**
Deploy to Alpha, make an account a teacher account and verify that the mail sent looks nice.
